go.sum

```diff
@@ -57,8 +57,8 @@ github.com/dgrijalva/jwt-go v3.2.0+incompatible/go.mod h1:E3ru+11k8xSBh+hMPgOLZm
 github.com/dgryski/go-sip13 v0.0.0-20181026042036-e10d5fee7954/go.mod h1:vAd38F8PWV+bWy6jNmig1y/TA+kYO4g3RSRF0IAv0no=
 github.com/dlclark/regexp2 v1.4.1-0.20201116162257-a2a8dda75c91 h1:Izz0+t1Z5nI16/II7vuEo/nHjodOg0p7+OiDpjX5t1E=
 github.com/dlclark/regexp2 v1.4.1-0.20201116162257-a2a8dda75c91/go.mod h1:2pZnwuY/m+8K6iRw6wQdMtk+rH5tNGR1i55kozfMjCc=
-github.com/dop251/goja v0.0.0-20220104154337-b93494d0c5d9 h1:QYh60R7QllMj/JKMtE2hCrwRqDbQq8vsUD63SDdVpHw=
-github.com/dop251/goja v0.0.0-20220104154337-b93494d0c5d9/go.mod h1:R9ET47fwRVRPZnOGvHxxhuZcbrMCuiqOz3Rlrh4KSnk=
+github.com/dop251/goja v0.0.0-20220110113543-261677941f3c h1:1XnAlcjYBdO7xsa2rhNB/BTztiu4cFKOxE+3brXVtG4=
+github.com/dop251/goja v0.0.0-20220110113543-261677941f3c/go.mod h1:R9ET47fwRVRPZnOGvHxxhuZcbrMCuiqOz3Rlrh4KSnk=
```

Comment:
This is due to apparently not running `go mo tidy` between rebases of https://github.com/grafana/k6/commit/478b9f53190f833b95cf55ca1cac87c36d1398ec, I remember doing it before the holidays, but I am pretty sure I forgot to it the many rebases this year :facepalm:

Reply:
Added CI check, although the ["error message"](https://github.com/grafana/k6/runs/4800502520?check_suite_focus=true) can use some work https://github.com/grafana/k6/pull/2333